src/core_plugins/metrics/public/kbn_vis_types/vis_controller.js

```diff
@@ -0,0 +1,49 @@
+import modules from 'ui/modules';
+import 'plugins/timelion/directives/refresh_hack';
+import 'ui/state_management/app_state';
+import '../directives/visualization';
+const app = modules.get('kibana/metrics_vis');
+
+app.controller('MetricsVisController', (
+  $scope,
+  $element,
+  Private,
+  timefilter,
+  getAppState,
+  $location
+) => {
+
+  // If we are in the visualize editor context (and not embedded) we should not
+  // render the visualizations. This is handled by the editor itself.
+  const embedded = $location.search().embed === 'true';
+  if (!embedded && $scope.vis._editableVis) {
```

Comment:
with that pr i put up you could probably also get rid of this as the visualize component wont be rendered at all (only your editor will)